lib/modules/platform/gitea/index.ts

```diff
@@ -10,13 +10,16 @@ import {
 } from '../../../constants/error-messages';
 import { logger } from '../../../logger';
 import type { BranchStatus } from '../../../types';
+import { deduplicateArray } from '../../../util/array';
 import { parseJson } from '../../../util/common';
 import * as git from '../../../util/git';
 import { setBaseUrl } from '../../../util/http/gitea';
+import * as p from '../../../util/promises';
```

Comment:
why are you using import star here instead of only import map function?

Reply:
This is how it was used everywhere else in the code but I can change to import only single function

Reply:
@viceice changed to import only map function